packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1992,7 +1992,7 @@ export class ContainerRuntime
 		// Note that the inner (non-proxy) delta manager is needed here to get the readonly information.
 		const connecting =
 			connected && !this._connected && !this.innerDeltaManager.readOnlyInfo.readonly;
-		if (connecting && this.blobManager.hasPendingOfflineUploads) {
+		if (connecting && this.blobManager.hasStashedPendingUploads) {
```

Comment:
at like 822 in the load method we wait on apply stashed ops, and initializebase container. i think we should move the waiting for blob upload around there, rather that here which will be called every connection